deps.ts

```diff
@@ -6,3 +6,4 @@ export { deferred } from "https://deno.land/std@0.130.0/async/deferred.ts";
 export type { Deferred } from "https://deno.land/std@0.130.0/async/deferred.ts";
 export * as b64 from "https://deno.land/std@0.130.0/encoding/base64.ts";
 export * as hex from "https://deno.land/std@0.130.0/encoding/hex.ts";
+export * as GeoJSON from "https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/master/types/geojson/index.d.ts";
```

Comment:
Can you download and save this type as another file in this repo? It's not good that this module to be depended on any external services, only deno.land is allowed

Reply:
> Can you download and save this type as an another file in this repo? It's not good that this module to be depended on any external services, only deno.land is allowed

Thanks for the review! I've just fixed it.